packages/SwingSet/src/kernel/collectionManager.js

```diff
@@ -339,7 +338,7 @@ export function makeCollectionManager(
         while (priorDBKey !== undefined) {
           assert(
             generationAtStart === currentGenerationNumber,
-            X`keys in store cannot be changed during iteration`,
+            X`keys in store cannot be added to during iteration`,
```

Comment:
Pre-existing issue, but it's the iterator that gets killed, right? `store.init()` works just fine. Maybe reword the error message to not make it sound like the addition is being blocked? Maybe `iterator terminated because a key was added to the store`?